### ISSUE TITLE
image-layout: Drop or reword the “blobs the manifest references” line

### ISSUE THREAD

**Issue by wking** [opencontainers/image-spec#427]
A [ref → descriptor example caption][1] gained “blobs the manifest references” wording with 2f24791, #136, but I don't see a direct relationship between descriptors and those deeper-ancestor blobs.  I think we either want to revert the changes 2f24791 made to this line or drop the line.  My preference would be to drop the line, because the other points covered by the sentence are already covered in other descriptor docs.

CC @runcom.

Spun off from #423, which is shifting the line.  I'm happy to file a PR after #423 lands.

[1]: https://github.com/opencontainers/image-spec/blame/5dbe4a342b6de21149f5c9f23c13b62ec1503082/image-layout.md#L67
